core/client/initializers/link-to.js

```diff
@@ -0,0 +1,22 @@
+var enhanceLinkTo = {
```

Comment:
I think we already have somewhere that we're reopening LinkView to play with its internals; can you double check that and consolidate this code together?